### ISSUE TITLE
Usage of a global in an API

### ISSUE THREAD

**Issue by rabin-goat** [cjagernath/google-slides-app#4]
Most of the server.js is written really well however the fundamental flaw I see here is the usage of a global to store the `points`

https://github.com/cjagernath/google-slides-app/blob/e9d1c15a788a0638a776a9f82e470dd4385dad15/server.js#L35

APIs are not **_EVER_** supposed to store state in the server, the state is always meant to be stored in the database. **_ALWAYS_**.

The usage of a global state with Async is what causes the code to behave unpredictably. This is because the server uses the first endpoint to store the points in a global state, and because of this the second API call can never work independently of the first one. They are _tightly coupled_. It is important to remember to strive for **predictable** and **functional** code. 

We can solve this by removing the usage of the global state `points` and having the front end make a `POST` call to the `create-presentation` endpoint that includes the `points` as part of the body, similar to the way `title` is passed.

**Comment by rabin-goat** [comment]
Another solution is to just create one endpoint for the whole process

**Comment by rabin-goat** [comment]
https://github.com/cjagernath/google-slides-app/blob/e9d1c15a788a0638a776a9f82e470dd4385dad15/src/App.js#L64-L69

```js
// should be
handleCreate(title,pointsArray);
```

In order to implement the solution outlined above you would have to pass the points data to the `handleCreate` function. Then handle the second endpoint using the data send from the front end

https://github.com/cjagernath/google-slides-app/blob/e9d1c15a788a0638a776a9f82e470dd4385dad15/src/App.js#L72-L81

```js
// should be
       body: JSON.stringify({ title, pointsArray }), 
```

**Comment by rabin-goat** [comment]
fixed in #6
